microservices/payment/src/services/payment-gateway/stripe.ts

```diff
@@ -877,31 +906,68 @@ class Stripe extends Abstract {
    * NOTES: How much end user will get after fees from transaction
    * 1. Stable unit - stable amount that payment provider charges
    * 2. Payment percent - payment provider fee percent for single transaction
-   */
-  private async getReceiverPaymentAmount(
-    totalUnitAmount: number,
+   * Fees calculation:
+   * 1. User pays fee
+   * totalAmount = 106$, receiverReceiver = 100, taxFee = 3, applicationFee = 3
+   * 2. Receiver pays fees
+   * totalAmount = 100$, receiverReceiver = 94, taxFee = 3, applicationFee = 3
+   */
+  private async getPaymentIntentFees({
+    entityCost,
+    feesPayer = TransactionRole.SENDER,
     applicationPaymentPercent = 0,
-  ): Promise<{
-    receiverUnitAmount: number;
+    additionalFeesPercent,
+    extraReceiverRevenuePercent = 0,
+  }: TGetPaymentIntentFeesParams): Promise<{
     paymentProviderUnitFee: number;
     applicationUnitFee: number;
+    userUnitAmount: number;
+    receiverUnitRevenue: number;
   }> {
-    const {
-      paymentFees: { stableUnit, paymentPercent },
-    } = await remoteConfig();
+    const { paymentFees } = await remoteConfig();
+
+    const { paymentPercent, stableUnit } = paymentFees || { paymentPercent: 2.9, stableUnit: 30 };
```

Comment:
```suggestion
    const { paymentPercent, stableUnit } = paymentFees!;
```

you already have a default value